sdcm/sct_config.py

```diff
@@ -1220,8 +1220,7 @@ class SCTConfiguration(dict):
              help="Number of iops for ebs type io2|io3|gp3"),
         dict(name="run_node_benchmarks", env="SCT_RUN_NODE_BENCHMARKS",
              type=boolean,
-             help="Flag for running db node benchmarks before the tests")
-
+             help="Flag for running db node benchmarks before the tests"),
```

Comment:
Addition of this missing comma must not be here. It should be added in the previous commit.
Need to reduce code line changes as much as possible when you do multi-commit PR.

Reply:
Fixed with rebase